devimint/src/federation/config.rs

```diff
@@ -51,9 +44,7 @@ pub fn attach_default_module_init_params(
     module_init_params.attach_config_gen_params(
         WalletInit::kind(),
         WalletGenParams {
-            local: WalletGenParamsLocal {
-                bitcoin_rpc: bitcoin_rpc.clone(),
-            },
+            local: EmptyGenParams {},
```

Comment:
Can we leave a tombstone comment here so nobody accidentally uses that field in the future and assumes it's always empty (not true for old federations)?